opslevel/resource_opslevel_team.go

```diff
@@ -289,12 +289,12 @@ func resourceTeamUpdate(d *schema.ResourceData, client *opslevel.Client) error {
 	if d.HasChange("group") {
 		return errors.New("groups are deprecated - create and update are disabled.")
 	}
-	if d.HasChange("parent") {
```

Comment:
What was happening before causing the parent to be unset:

1. The parent is set
2. It has no change
3. Therefore `input.ParentTeam = nil`, because it is a pointer to an identifier input without `omitempty`.

Also removed `GetOk` because it's not necessary here.